devops/config/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Prepends the ticket number to the commit message if the branch name has the expected format.
```

Comment:
I don't think we need this for this project, it does not work anyway as expected, since Github won't automatically make Azure ticket numbers clickable, only if they are written in the format of `AB#TICKET_NUMBER`